test/e2e/auth/audit.go

```diff
@@ -31,14 +31,15 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/wait"
 	auditinternal "k8s.io/apiserver/pkg/apis/audit"
-	"k8s.io/apiserver/pkg/apis/audit/v1"
+	auditv1 "k8s.io/apiserver/pkg/apis/audit/v1"
```

Comment:
Why do you need this change, it seems cosmetic.

Reply:
It is cosmetic but @andrewsykim requested it and I believe it is consistent with some of the other references to this package.

Reply:
TL;DR when doing refactoring work, decouple these changes as a separate PR of commit so it's clear to the approver what the intent of your PR is.  Each commit should do just (1) thing.